code/modules/reagents/Chemistry-Machinery.dm

```diff
@@ -220,7 +220,7 @@
 		user << "Something is already loaded into the machine."
 		return
 
-	if(istype(B, /obj/item/weapon/reagent_containers/glass) || istype(B,/obj/item/weapon/reagent_containers/food))
+	if(istype(B, /obj/item/weapon/reagent_containers/glass) || istype(B, /obj/item/weapon/reagent_containers/food/drinks/drinkingglass))

```

Comment:
Bah, I saw that the chem master checked for `/drinkingglass` and thought, "Yeah, that sounds right."